app/helpers/invitation_codes_helper.rb

```diff
@@ -6,13 +6,13 @@ def invite_hidden_tag(invite)
   end
 
   def invite_link(invite_code)
-    text_field_tag :invite_code, invite_code_url(invite_code), :readonly => true
+    text_field_tag :invite_code, invite_code_url(invite_code), :class => 'form-control', :readonly => true
```

Comment:
We need to add that class kind of everywhere. Is it possible to overwrite this helper? Otherwise we might want to add a new helper that does that for us.

@jhass ping.